test/e2e_test.go

```diff
@@ -78,6 +78,10 @@ func (e *e2e) TestSecurityProfilesOperator() {
 			"Seccomp: Delete profiles",
 			e.testCaseDeleteProfiles,
 		},
+		{
+			"Seccomp: Metrics",
+			e.testCaseMetrics,
+		},
```

Comment:
oho! I just noticed, the re-deploy test needs to go on spot 4 in the array. See line 128.

Reply:
Oops, I changed it to `5`.